App/screens/Login.js

```diff
@@ -1,87 +1,93 @@
 import React from 'react';
-import { Text, TextInput, KeyboardAvoidingView, ScrollView, Button } from 'react-native';
+import {Text, TextInput, KeyboardAvoidingView, ScrollView, Button} from 'react-native';
 import { connect } from 'react-redux';
 import styles from '../theme/ApplicationStyles';
-
-// this is to not remember password, skip it :)
+import OfflineModal from '../components/Offline';
 import {email, password} from '../configs/sensitive';
 import { login } from '../redux/auth';
+import * as Animatable from 'react-native-animatable';
```

Comment:
This lib is missing in package.json

Reply:
It's a dependency of the other, but I will explicitly add it.